docs/.vitepress/config.mts

```diff
@@ -32,7 +32,7 @@ export default defineConfig({
             ]
           },
           { text: 'Development Guide', link: '/Development_Guide' },
-          { text: 'C++ Plugins Guide', link: '/Cpp_Plugins_Guide' }
+          { text: 'C++ Plugins Guide', link: '/Plugins_Guide' }
```

Comment:
Sidebar still says "C++ Plugins Guide" while the title of the document is just "Plugins Guide"

Reply:
oops fixed, thanks

Reply:
fixed